src/client-ocurl/opentelemetry_client_ocurl.ml

```diff
@@ -348,7 +348,9 @@ end = struct
         Queue.clear local_q;
 
         if !must_flush_all then (
-          if Batch.len batches.metrics > 0 then send_metrics ();
+          if Batch.len batches.metrics > 0 || not (AList.is_empty gc_metrics)
+          then
+            send_metrics ();
```

Comment:
This covers only the `flush_all` case right? I think we should cover the other case below too

https://github.com/imandra-ai/ocaml-opentelemetry/pull/55/files#diff-268847e82615141defe96cc7afc6bdf2dea64addd7f56052ec6124d7832fc9c3R358-R359